src/components/ebay-video/README.md

```diff
@@ -17,13 +17,19 @@ For resizing, `ebay-video` supports fixed width or variable width. If no width i
 
 Name | Type | Stateful | Required | Description
 --- | --- | --- | --- | ---
-`src` | String | No | Yes | The url path for the playlist MPD or M3U8 to use. If format is set to auto, will detect format based on file extension
-`format` | String | No | No | Either `auto`, `mpd` or `m3u8`. Default is auto. The format to use. If auto, will try to detect based on file name. If mpd will trigger downloading dashjs from given CDN.
-`cdn-url` | String | No | No | The full url to point to where to download dashjs. This is only used to override the default CDN path.
-`cdn-version` | String | No | No | If using the default CDN path, you can override what version of dashjs to use.
+`src` | String | No | Yes | The url path for the playlist MPD or M3U8 to use.
+`thumbnail` | String | No | No | The url path for the video thumbnail
+`cdn-url` | String | No | No | The full url to point to where to download shaka. This is only used to override the default CDN path.
+`cdn-version` | String | No | No | If using the default CDN path, you can override what version of shaka to use.
```

Comment:
maybe we could be more specific with this name - it could be 
`shaka-version-cdn` or something like that. 

Just a suggestion though

Reply:
Good comment, I think I want to make an option to change possibly, but not sure.
We can revisit it